docs/src/components/ChunkedSnippetExperimental.astro

```diff
@@ -0,0 +1,231 @@
+---
+import  fs  from "node:fs/promises"
+import path from "node:path"
+import { execSync } from 'node:child_process';
+import prettier from 'prettier';
+import { Code } from 'astro:components';
+import { CodeBlock } from '@interledger/docs-design-system'
+const {source, chunk, version, l} = Astro.props;
+import * as phpPlugin from '@prettier/plugin-php'
+
+const gitRoot = execSync('git rev-parse --show-toplevel').toString().trim();
+const snippetsPath = path.join(gitRoot, 'snippets');
+
+const getChunk = async () => {
+  const data = await fs.readFile(path.join(snippetsPath, version, l, source), {encoding: "utf8"})
+  return data
+}
```

Comment:
This is the only code change that happened in this file - how we retrieve the source: `readFile` instead of a HTTP request.